tests/sentry/ratelimits/utils/test_get_rate_limit_value.py

```diff
@@ -49,10 +49,42 @@ class TestEndpoint(Endpoint):
             20, 4
         )
 
+    def test_inherit(self):
+        class ParentEndpoint(Endpoint):
+            rate_limits = {"GET": {RateLimitCategory.IP: RateLimit(100, 5)}}
+
+        class ChildEndpoint(ParentEndpoint):
+            rate_limits = {"GET": {}}
+
+        assert get_rate_limit_value("GET", ChildEndpoint, RateLimitCategory.IP) == RateLimit(100, 5)
+
+    def test_multiple_inheritance(self):
```

Comment:
Could you add the case we talked about earlier, where an endpoint inherits multiple parent endpoints?